bot/exts/utilities/bookmark.py

```diff
@@ -25,6 +23,49 @@
 )
 
 
+class LinkTargetMessage(discord.ui.View):
+    """The button that relays the user to the bookmarked message."""
+
+    def __init__(
+        self,
+        target_message: discord.Message,
+    ):
+        super().__init__()
+        self.add_item(discord.ui.Button(label="View Message", url=target_message.jump_url))
+
+
+class SendBookmark(discord.ui.View):
+    """The button that sends the bookmark to other users."""
+
+    def __init__(
+        self,
+        bookmark_function: Callable[[discord.TextChannel, discord.Member, discord.Message, str], None],
+        author: discord.Member,
+        channel: discord.TextChannel,
+        target_message: discord.Message,
+    ):
+        super().__init__()
+
+        self.bookmark_function = bookmark_function  # The function is the 'action_bookmark'
+        self.clicked: list[int] = [author.id]
```

Comment:
```suggestion
        self.clicked = [author.id]
```
This type-hint is unnecessary.